test/lev-report/charts/HourlyUsage.test.js

```diff
@@ -51,22 +50,6 @@ describe('HourlyUsage', () => {
 					.toJSON();
 			expect(tree).toMatchSnapshot();
 		});
-
-		afterAll(() => {
-			if (process.env.VISUALISE) {
-				const html = `<html>
-			<head>
-					<title>Component Preview</title>
-					<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-			</head>
-			<body style="margin:0;">
-					<div id="app">
-						 ${renderToString(<HourlyUsage traces={traces}/>)}
-					</div>
-			</body>
-			</html>`;
-				require('fs').writeFileSync('test/lev-report/charts/__snapshots__/HourlyUsage.html', html);
-			}
-		});
+		require('../../visualise')(<HourlyUsage traces={traces}/>, 'test/lev-report/charts/__snapshots__/HourlyUsage.html');
```

Comment:
Could we move the require to the top?

i.e.
```js
import visualise from '../../visualise';
```

Reply:
Yes sure. Fixed